ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/ros2/RDXROS2MultiBodyGraphic.java

```diff
@@ -1,34 +0,0 @@
-package us.ihmc.rdx.ui.graphics.ros2;
-
-import us.ihmc.rdx.ui.graphics.RDXMultiBodyGraphic;
-import us.ihmc.ros2.ROS2Topic;
-
-public class RDXROS2MultiBodyGraphic extends RDXROS2SingleTopicVisualizer
```

Comment:
This wasn't being extended anywhere or used directly. Seems useless to me, so collapsed it.